base/doc/usrguide.tex

```diff
@@ -1216,8 +1216,21 @@ \section{Case changing}
   \item Lowercasing
   \item Titlecasing (only applies for the start of the input)
 \end{itemize}
+
 The command \cs{DeclareCaseChangeEquivalent} provides a way to substitute a
 command by an alternative version when it is found inside a case changing
-situation.
+situation. There are three commands for customising the case changing of
+codepoints
+\begin{decl}
+  |\DeclareLowercaseMapping| \oarg{locale} \arg{codepoint} \arg{output} \\
+  |\DeclareTitlecaseMapping| \oarg{locale} \arg{codepoint} \arg{output} \\
+  |\DeclareUppercaseMapping| \oarg{locale} \arg{codepoint} \arg{output}
+\end{decl}
+All three take a \meta{codepoint} (as an integer expression) and will
+result in the \meta{output} being produced under the appropriate case changing
+operation. The optional \meta{locale} can be given if the mapping should only
+apply to a specific one: this is given in BCP-47 format.
+
```

Comment:
I suggest to add a useful link for BCP-47 maybe https://en.wikipedia.org/wiki/IETF_language_tag and also give two examples, one with and one without the optional locale

Reply:
OK will do